### DISCUSSION TITLE
Blocked newly installed app

### DISCUSSION THREAD

**Discussion by iulko** [celzero/rethink-app#927] [Q&A]
While doing no new install or anything like that I got an notification that Rethink blocked newly installed app: 0

What would the 0 be?

**Comment by ignoramous** [comment]
It is `Android` itself.

A notification means that's the first time Rethink is "seeing" it *make* connections.

As you may guess, `Android` cannot be installed / uninstalled... but yeah, that's how Rethink treats such "apps" that aren't really apps but appear to it as so.

Generally speaking, the Android OS assigns *ownership* of outgoing connections to itself, erroneously sometimes. For example, I find the OS often attribute outgoing connections from Firefox to itself aka user `0` aka `Android`.
